deeppavlov/dataset_readers/mltrack_reader.py

```diff
@@ -0,0 +1,247 @@
+# Copyright 2017 Neural Networks and Deep Learning lab, MIPT
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, softwaredata
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import csv
+import re
+from typing import Dict, List, Tuple
+
+from deeppavlov.core.commands.utils import expand_path
+from deeppavlov.core.common.registry import register
+from deeppavlov.core.data.dataset_reader import DatasetReader
+
+
+@register("mltrack_reader")
+class MLtrackReader(DatasetReader):
+    """The class to read the MLtrack dataset from files.
+
+    Please, see https://contest.yandex.ru/algorithm2018
```

Comment:
This url doesn't really clarify things

Reply:
Please provide a more detailed description of the class

Reply:
I've changed url and extended the description.